### PR TITLE
doc: fixing encapsulation

### PR DESCRIPTION
**Pull request by zekth**
Fixing a minor issue in the encapsulation doc.

### CODE REVIEW

File: docs/Encapsulation.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ fastify.register(async function authenticatedContext (childServer) {
   childServer.register(require('fastify-bearer-auth'), { keys: ['abc123'] })
 
   childServer.route({
-    path: '/one',
+    url: '/one',
```
Comment (jsumners) [inline docs/Encapsulation.md:50]: `path` is a valid alias.

File: docs/Encapsulation.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ fastify.register(async function authenticatedContext (childServer) {
...
     method: 'GET',
-    handler (request, response) {
+    handler: (request, response) => {
```
Comment (jsumners) [inline docs/Encapsulation.md:52]: I very specifically did not use an arrow function here. I used shorthand method notation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions#description

**Review by jsumners — CHANGES_REQUESTED** [review]
I could be convinced to change the `path` properties to `url` properties, but I cannot accept the methods being arrow functions.

File: docs/Encapsulation.md (hunk 1 above)
Comment (zekth) [reply docs/Encapsulation.md:50]: My bad, so seems like typing is wrong: https://github.com/fastify/fastify/blob/master/types/route.d.ts#L111

**Pull request closed without merging**